airflow/docker_image.go

```diff
@@ -36,6 +36,12 @@ const (
 
 var errGetImageLabel = errors.New("error getting image label")
 
+func realGetDockerClient() (client.APIClient, error) {
+	return client.NewClientWithOpts(client.FromEnv)
+}
+
+var getDockerClient func() (client.APIClient, error) = realGetDockerClient
```

Comment:
```suggestion
var getDockerClient = func() (client.APIClient, error) {
	return client.NewClientWithOpts(client.FromEnv)
}
```
nit: just reducing an extra function declaration and figuring out a name for that 🙂